libs/libvtrutil/src/vtr_geometry.tpp

```diff
@@ -81,10 +81,11 @@ Rect<T>::Rect(Point<T> bottom_left_val, Point<T> top_right_val)
 }
 
 template<class T>
-Rect<T>::Rect(Point<T> bottom_left_val, T size)
-    : bottom_left_(bottom_left_val)
-    , top_right_(bottom_left_val.x() + size,
-                 bottom_left_val.y() + size) {
+template<typename U, typename std::enable_if<std::is_integral<U>::value>::type...>
```

Comment:
The `enable_if` etc stuff probably deserves a comment

Reply:
Done.